README.md

```diff
@@ -83,7 +83,7 @@ We bump `Major.Minor.Patch` versions following this guidance:
 `Major`: (we've never done this)
 
 - Significant backwards-incompatible changes to the public API that would require rewriting existing application code.
-- Some examples of backwards-incompatible changes we might someday consider for a Major release are at [`ROADMAP.md`](ROADMAP.md).
+- See ROADMAP.md or examples of backwards-incompatible changes we might someday consider for a Major release.
```

Comment:
After some thought, I think we should delete this bullet item and drop the reference to ROADMAP.md altogether (and we shouldn't ship that file in the gem). It's a very informal wishlist and I don't want people to take it too seriously.

Reply:
Fixed.